certified-connectors/Africa's Talking Airtime/apiDefinition.swagger.json

```diff
@@ -0,0 +1,303 @@
+                  "description": "responses"
+                },
+                "totalAmount": {
+                  "type": "string",
+                  "description": "totalAmount"
+                },
+                "totalDiscount": {
+                  "type": "string",
+                  "description": "totalDiscount"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Send Airtime",
+        "description": "Send airtime to multiple recipients",
+        "operationId": "SendAirtime",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "username": {
+                  "type": "string",
+                  "description": "Your application's username.",
+                  "title": "Username",
+                  "x-ms-visibility": "important"
+                },
+                "recipients": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "phoneNumber": {
+                        "type": "string",
+                        "description": "Phone number to be topped up in international format (e.g +234811222333)",
+                        "title": "Phone Number",
+                        "x-ms-visibility": "important"
+                      },
+                      "amount": {
+                        "type": "string",
+                        "description": "The value of airtime to send together with the currency code (e.g. KES 10.00).",
+                        "title": "Amount",
+                        "x-ms-visibility": "important"
+                      }
+                    },
+                    "required": [
+                      "amount",
+                      "phoneNumber"
+                    ]
+                  },
+                  "description": "recipients"
+                }
+              },
+              "x-ms-visibility": "important",
+              "required": [
+                "recipients",
+                "username"
+              ]
+            },
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/query/transaction/find": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200",
```

Comment:
@crispinkoech Can you please provide a valid description for the "200" response code instead of "200"